gpm_api/tests/test_dataset/test_granule_files.py

```diff
@@ -38,17 +38,18 @@ def test_open_granule_on_real_files(tmp_path):
     ├── cut
     │   ├── V7/RS/1A-GMI
     │   │   └── 1A.GPM.GMI.COUNT2021.20140304-S223658-E000925.000082.V07A.HDF5
-    ├── processed
-    │   ├── V7/RS/1A-GMI
-    │       ├── S1.nc
-    │       ├── S2.nc
-    │       ├── S4.nc
-    │       └── S5.nc
-    └── ...
+    │   └── ...
+    └── processed
+        ├── V7/RS/1A-GMI
+        │    ├── S1.nc
+        │    ├── S2.nc
+        │    ├── S4.nc
+        │    └── S5.nc
+        └── ...
     """
 
-    granules_dir_path = os.path.join(_root_path, "gpm_api", "tests", "data", "granules")
-    granules_dir_path = os.path.join("/home/ghiggi/GPM_TEST_DATA_DEMO")
+    script_path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
For sake of clarity, and also to facilitate debugging, I am used to always define this type of paths from the `_root_path` of the gpm_api repository: `from gpm_api import _root_path`
 Are you against this usage?

Reply:
No you are right! I probably got confused by the home path and deleted everything...